aiida/orm/implementation/querybuilder.py

```diff
@@ -126,6 +127,52 @@ def get_filter_expr_from_attributes(cls, operator, value, attr_key, column=None,
         """
         pass
 
+    @classmethod
+    def get_filter_expr_from_column(cls, operator, value, column):
+        """
+        A method that returns an valid SQLAlchemy expression.
+
+        :param operator: The operator provided by the user ('==',  '>', ...)
+        :param value: The value to compare with, e.g. (5.0, 'foo', ['a','b'])
+        :param column: an instance of sqlalchemy.orm.attributes.InstrumentedAttribute or
+
+        :returns: An instance of sqlalchemy.sql.elements.BinaryExpression
+        """
+        # Label is used because it is what is returned for the
```

Comment:
I think this comment is outdated, because it refers to the old `DbCalcState` table that has been removed. The `state` is now a regular attribute. This might make the conditional below easier as we can remove `Label` but I am not sure.